src/Analysis/AnalysisApp.jsx

```diff
@@ -110,12 +111,28 @@ class AnalysisApp extends React.Component {
     });
   }
 
+  handleFullscreenButtonClick = () => {
+    this.setState({
+      analysisIsFullscreen: !this.state.analysisIsFullscreen,
+    });
+  }
+
   render() {
     const { job, params } = this.props;
     const { loaded, app, results } = this.state;
     if (!app) {
       return <Spin size='large' tip='Loading App...' />;
     }
+
+    const fullscreenButton = (
```

Comment:
this button should only be visible when the `appContent` is actually an iframe app